operations/boot_orchestration/Ascertain_Which_BOS_Session_Booted_A_Node.md

```diff
@@ -0,0 +1,87 @@
+# Ascertain Which BOS Session Booted a Node
+
+This guide is split into BOS Version 1 (V1) and BOS Version 2 (V2) sections
+because the procedures are different for the two different versions.
+
+## BOS Version 1 (V1)
+
+To determine which BOS Session booted or rebooted a node, query the node's
+kernel boot parameters. They contain a string 'bos_session_id' which identifies
+which BOS session booted or rebooted the node. Then, use this BOS Session ID
+to describe the BOS Session, which identifies the BOS Session template used.
+
+### Query the Node
+
+From a management node (master or worker), ssh to the node in question.
+(`ncn-mw#`) ssh <node's xname>
+('<node>#') cat /proc/cmdline
```

Comment:
There are many ways to do this, but one way to get just the BOS session ID would be:

```
ncn-m001:~ # ssh x1000c0s2b0n0 "cat /proc/cmdline" | awk -v RS=" " -F "=" '{if ($1 == "bos_session_id") { print $2; }}'
4b6744ee-837f-4f60-9051-897aed6c7623
```

Reply:
This is a better way to do what I was attempting to do. I will adopt this method.